### PR TITLE
Consolidate on ip instead of mix of ip and ifconfig for ip4 and ip6 detection

### PR DESCRIPTION
**Pull request by skahler-pivotal**
Using ifconfig for ipv4 detection and ip show for ipv6 detection doesn't make sense when it can be consolidated down to using one tool possibly removing an installation dependency

### CODE REVIEW

**Comment by ivannovick** [comment]
so what are we changing?

Is it from ifconfig to ip -4 ?

ip -4 doesn't work on my linuxbox, did i get the right syntax to run the command manually?

**Comment by skahler-pivotal** [comment]
The change would be moving from ifconfig to "ip -4 address show" which essentially the v4 version of what is currently used to detect v6 addresses

**Comment by asimrp** [comment]
Look good to me.  Are there any subtleties with `ip` on certain Linux systems?  E.g. `ping6` behaves a bit differently on some CentOS platforms as compared to other Linux's.  #171 has an example.

Please try to avoid mixing whitespace only changes with legitimate changes.  One more request - please collapse all those commits into one before pushing to master.

**Comment by skahler-pivotal** [comment]
Do you want me to pull out this request and create another with everything merged?

I've tested with centos6 and centos7 and I'm working on building up other platforms to test on, I do not think there is any difference in other linux platforms. Which other platforms do we want to validate work?

**Comment by ivannovick** [comment]
@skahler-pivotal We should verify centos 6, centos 7, SuSE 11 SP3, SuSE 12, and latest Debian.

That should be good.

**Comment by asimrp** [comment]
On Tue, Dec 29, 2015 at 7:12 AM, Scott Kahler <notifications@github.com>
wrote:

> Do you want me to pull out this request and create another with everything
> merged?
>
No need for new request.  Just squash the commits into one before pushing
to master.

**Comment by skahler-pivotal** [comment]
Okay got it squashed now

**Comment by ascherbaum-pivotal** [comment]
> centos 6, centos 7, SuSE 11 SP3, SuSE 12, and latest Debian

Added this list to the wishlist for the buildfarm client.

File: gpMgmt/bin/lib/gp_bash_functions.sh [hunk 1]
```diff
@@ -2000,12 +2005,11 @@ case $OS_TYPE in
...
 		NOLINE_ECHO=$ECHO
 		DEFAULT_LOCALE_SETTING=en_US.utf-8
-        PING6=`findCmdInPath ping6`
+        	PING6=`findCmdInPath ping6`
 		PING_TIME="-c 1"
 		DF="`findCmdInPath df` -P"
 		DU_TXT="-c" ;;	
-	freebsd ) IFCONFIG_TXT=""
-		PS_TXT="ax"
+	freebsd ) PS_TXT="ax"
```
Comment (danielgustafsson) [inline gpMgmt/bin/lib/gp_bash_functions.sh]: What does it mean for FreeBSD compatibility that `IPV{4|6}_ADDR_LIST_CMD` aren't defined?

Comment (skahler-pivotal) [inline gpMgmt/bin/lib/gp_bash_functions.sh]: Good catch, I think that means they will break and not do the lookup. I'll add it in

**Pull request merged**